packages/store/tests/issues/issue-1603-no-type-property-on-the-action.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { TestBed } from '@angular/core/testing';
+import { NgxsModule, State, Action, Store, Actions, ofActionDispatched } from '@ngxs/store';
+
+describe('https://github.com/ngxs/store/issues/1603', () => {
```

Comment:
PS. I would prefer a description of the scenario here (we already have the issue reference number in the file path)

Reply:
Okay, will upd.